test/governance.js

```diff
@@ -0,0 +1,691 @@
+/* eslint-disable no-undef */ // Avoid the linter considering truffle elements as undef.
+
+const BigNumber = web3.BigNumber
+const {
+  expectThrow
+} = require('openzeppelin-solidity/test/helpers/expectThrow')
```

Comment:
We have these in `kleros-interaction`.

Reply:
Hmm, why don't we just let OpenZeppelin maintain them for us?

Reply:
Then we should change it everywhere else as well. I'll make an issue.